content/_changelogs/10.7.0.md

```diff
@@ -0,0 +1,39 @@
+## 10.7.0
+
+_Released 8/30/2022_
+
+
+**Features:**
+
+- Re-introducing the experimental [Cypress Studio feature](https://docs.cypress.io/guides/references/cypress-studio). Cypress Studio provides a visual way to generate tests within Cypress by recording interactions against the application under test. Cypress Studio must be enabled via the `experimentalStudio` flag and is currently not supported in component tests. Addresses [#23461](https://github.com/cypress-io/cypress/issues/23461)
+- Adds [Svelte](https://svelte.dev/) component testing support. Addresses [#23431](https://github.com/cypress-io/cypress/issues/23431) and [#23432](https://github.com/cypress-io/cypress/issues/23432).
+- New informational banners will display instructions to assist with connecting projects to the Cypress Dashboard and recording a first test run. Users who don't plan to use the Cypress Dashboard can dismiss the banners to not see them in the future. Addresses [#22933](https://github.com/cypress-io/cypress/issues/22933), [#22934](https://github.com/cypress-io/cypress/issues/22934), [#22798](https://github.com/cypress-io/cypress/issues/22798), and [#22935](https://github.com/cypress-io/cypress/issues/22935)
+- Vue.js projects with a custom spec pattern configuration can now use the "Create Spec from Component" capability introduced in v10.5.0. Addresses [#23298](https://github.com/cypress-io/cypress/issues/23298)
```

Comment:
This one and its PR also aren't inside a release -- @tbiethman we should add the release to the issue when it's missing, right?

Reply:
Tagged issue, updating link to point to issue instead of PR